src/pages/home/report/ReportActionItemDate.tsx

```diff
@@ -1,31 +1,24 @@
 import React, {memo} from 'react';
-import type {OnyxEntry} from 'react-native-onyx';
-import {withCurrentDate} from '@components/OnyxProvider';
+import {useOnyx} from 'react-native-onyx';
 import Text from '@components/Text';
 import useLocalize from '@hooks/useLocalize';
 import useThemeStyles from '@hooks/useThemeStyles';
+import ONYXKEYS from '@src/ONYXKEYS';
 
-type ReportActionItemDateOnyxProps = {
-    /**
-     * UTC timestamp for when the action was created.
-     * This Onyx prop is hooked to the current date so that relative times can update when necessary
-     * e.g. past midnight.
-     */
-    // eslint-disable-next-line react/no-unused-prop-types
-    currentDate: OnyxEntry<string>;
-};
-
-type ReportActionItemDateProps = ReportActionItemDateOnyxProps & {
+type ReportActionItemDateProps = {
     created: string;
 };
 
 function ReportActionItemDate({created}: ReportActionItemDateProps) {
     const {datetimeToCalendarTime} = useLocalize();
     const styles = useThemeStyles();
 
+    // It is used to force re-render of component that display relative time, ensuring they update correctly when the date changes (e.g., at midnight).
+    useOnyx(ONYXKEYS.CURRENT_DATE, {canBeMissing: false});
```

Comment:
Let's either `useContext` or remove the current date context. No point in having the context if it isn't used

Reply:
I agree, context removed